dnsupdate.sh

```diff
@@ -2,14 +2,25 @@
 
 USERNAME="" #INWX USERNAME
 PASSWORD="" #INWX PASSWORD
-DNSID="" #DNS Entry ID
+DNSIDv4="" #DNS Entry ID
+DNSIDv6="" #DNS Entry ID
 APIHOST="https://api.domrobot.com/xmlrpc/"
-OLDIP=$(cat old.ip)
-NEWIP=$(curl -s curlmyip.com)
+OLDIPv4=$(cat old.ipv4)
+OLDIPv6=$(cat old.ipv6)
+NEWIPv4=$(curl -s curlmyip.com)
+NEWIPv6=$(curl -s curlmyip6.com)
```

Comment:
What about moving the ```NEWIPv6``` assignment down to line 19? If only IPv4 is used, we do not need to send the second request. Same idea for the ```OLDIPv6``` line.